Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -281,8 +281,9 @@ GDCMImageIO::Read(void * pointer)
 
   itkAssertInDebugAndIgnoreInReleaseMacro(gdcm::ImageHelper::GetForceRescaleInterceptSlope());
 // Only available in newer versions
-#if (!defined(ITK_USE_SYSTEM_GDCM) || \
-     ((GDCM_MAJOR_VERSION == 3 && GDCM_MINOR_VERSION == 0 && GDCM_BUILD_VERSION > 23) || GDCM_MAJOR_VERSION > 3))
+#if (!defined(ITK_USE_SYSTEM_GDCM) ||                                                    \
+     ((GDCM_MAJOR_VERSION == 3 && GDCM_MINOR_VERSION == 0 && GDCM_BUILD_VERSION > 23) || \
+      (GDCM_MAJOR_VERSION == 3 && GDCM_MINOR_VERSION >= 1) || GDCM_MAJOR_VERSION > 3))
```

Comment:
I think it would be slightly more elegant to do `> 0`  instead of `>= 1`, for the sake of consistency. No show-stopper, of course!

Reply:
Updated to `> 0`